docs/5.x/cluster.md

```diff
@@ -221,6 +221,51 @@ $ curl -sk https://localhost:7575 | python -m json.tool
 
 In this case the response HTTP status code will be `503 Service Unavailable`.
 
+### Cluster Status History
+
+Running `gravity status history` will display a history of changes to the 
+cluster status. 
+
+Example output may looks something like the following:
+
+```bsh
+$ gravity status history
+2020-02-18T01:36:07Z [Node Degraded]     node=node-1
+2020-02-18T21:36:11Z [Node Degraded]     node=node-2
+2020-02-18T21:36:25Z [Node Degraded]     node=node-3
+2020-02-18T21:36:56Z [Probe Succeeded]   node=node-1 checker=node-status
+2020-02-18T21:36:58Z [Probe Succeeded]   node=node-2 checker=node-status
+2020-02-18T21:36:58Z [Probe Succeeded]   node=node-2 checker=time-drift
+2020-02-18T21:36:59Z [Probe Succeeded]   node=node-3 checker=node-status
+2020-02-18T21:37:07Z [Probe Succeeded]   node=node-1 checker=kube-apiserver
+2020-02-18T21:37:07Z [Node Recovered]    node=node-1
+2020-02-18T21:37:08Z [Probe Succeeded]   node=node-2 checker=kube-apiserver
+2020-02-18T21:37:08Z [Node Recovered]    node=node-2
+2020-02-18T21:37:11Z [Probe Succeeded]   node=node-3 checker=kube-apiserver
+2020-02-18T21:37:11Z [Node Recovered]    node=node-3
+```
+
+Here's an example of how to view the history remotely via `tsh`:
+
+```bsh
+$ tsh --cluster=production ssh admin@node gravity status history
+```
+
+The `gravity status history` command is an additional tool to help debug issues
+with a Cluster. The `gravity status` command only displays the current status
```

Comment:
nit: cluster (non-capitalized)

As a side note, this reads as following: we have `gravity status` which is not useful enough hence we have `gravity status history`. Shouldn't `gravity status` instead have been made useful? I know this is probably too late of a discussion, but including interesting historical developments in `gravity status` so it is useful would've been more helpful (like automatically including the last events leading to `NodeDegraded` if the node is still unhealthy, etc.).
This could be something to look into later.

Reply:
Good point. We can leave out the section pointing out the limitations of `gravity status` and just focus on what the new `history` sub-command provides.